code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -87,22 +93,23 @@
 	balloon_alert(user, "analyzing vitals")
 	playsound(user.loc, 'sound/items/healthanalyzer.ogg', 50)
 
+	var/readability_check = user.can_read(src) && !user.is_blind()
 	switch (scanmode)
 		if (SCANMODE_HEALTH)
-			healthscan(user, M, mode, advanced)
+			if(readability_check)
+				healthscan(user, M, mode, advanced)
 			last_scan_text = healthscan(user, M, mode, advanced, tochat = FALSE)
```

Comment:
you can simplify this to ``last_scan_text = healthscan(user, M, mode, advanced, tochat = readability_check)`` if you make healthscan always return the render_list (no reason not to do that), useful in case something hooks up an action (beyond additional text) to COMSIG_LIVING_HEALTHSCAN

Reply:
tooltips? Wonder how those look on paper, probably not very good (or possibly just broken), gotcha